test/jdk/sun/security/mscapi/EncodingMutability.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2005, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should only have the initial year the code was created and the last year that the code is updated, so in this case it should just have 2023.